README.md

```diff
@@ -10,7 +10,7 @@ This library offers a simple contract to use the biometry across Android, iOS an
    
 1. Get an `IBiometryService` instance.
 
-   `IBiometryService` is implemented by `BiometryService`.
+   `IBiometryService` is implemented by the abstract class `BaseBiometryService`, and `BaseBiometryService` is inherited by the concrete class `BiometryService`.
```

Comment:
I don't think think level of technical details is relevant in the getting started section.

Reply:
Should I remove the changes?

Reply:
Yes you should remove this specific change in the documentation.

Reply:
done